get-started/codelab/index.md

```diff
@@ -857,7 +857,7 @@ which pushes the route to the Navigator's stack.
 
 <li markdown="1"> The builder property returns a Scaffold,
     containing the app bar for the new route, named
-    "Saved Suggestions." The body of the new route
+    "Saved Suggestions". The body of the new route
```

Comment:
The Google Style Guide uses the American convention of putting the period inside the quotes: https://developers.google.com/style/quotation-marks